### PR TITLE
Add support for AWS SNS for notifications

### PR DESCRIPTION
**Pull request by marcsanfacon**

### CODE REVIEW

File: setup/pi/setup-teslausb [hunk 1]
```diff
@@ -455,6 +461,7 @@ else
 fi
 
 update_package_index
+install_python_packages
```
Comment (marcone) [inline setup/pi/setup-teslausb]: can this be made conditional on using sns? E.g. by moving the call to install_python_packages into configure_sns ?

Comment (marcsanfacon) [reply setup/pi/setup-teslausb]: I see that configure.sh step can be skipped, so I propose to add a condition that SNS is configured directly in setup-teslausb to install the python packages. What do you think?

Comment (marcone) [reply setup/pi/setup-teslausb]: configure.sh is only skipped when archiving is not configured, in which case no notifications will be sent anyway. If SNS needs python3/pip/boto3, it seems cleanest to install those in the script that configures SNS.

**Comment by marcsanfacon** [comment]
I move the install_python_packages with configure_sns.

**Comment by marcone** [comment]
I squashed the two changes together, fixed some whitespace-error and merged it to the main-dev branch: https://github.com/marcone/teslausb/commit/5683c4d02cfd70b06170fd94d1eb4d8e8f3d176f

Thanks!

**Pull request closed without merging**